project.clj

```diff
@@ -7,19 +7,19 @@
                  [marick/clojure-commons "3.0.0" :exclusions [org.clojure/clojure]]
                  ;; structural-typing currently broken with specter 0.13
                  ;; [marick/structural-typing "2.0.4" :exclusions [org.clojure/clojure org.clojure/clojurescript]]
-                 [org.clojure/math.combinatorics "0.1.3"]
+                 [org.clojure/math.combinatorics "0.1.4"]
```

Comment:
Midje 1.9.0 drops support for Clojure 1.6:

https://github.com/marick/Midje/blob/master/CHANGELOG.md#190-alpha6---2016-10-26

So this is fine